rom/dev/src/rom.ld

```diff
@@ -19,7 +19,8 @@ ENTRY(_start)
 ROM_ORG          = 0x00000000;
 ICCM_ORG         = 0x40000000;
 DCCM_ORG         = 0x50000000;
-DATA_ORG         = 0x50004000;
+/* Avoid overlap with other long-lived DCCM structures */
+DATA_ORG         = 0x50004400;
```

Comment:
not related to this PR, but create an issue to generate rom.ld from memory_layout.rs information.

Reply:
This change is no longer needed. Removing.